app/controllers/question.js

```diff
@@ -1,33 +1,71 @@
 const appInsights = require('app-insights');
 const express = require('express');
+const paths = require('paths');
+const { answerValidation } = require('app/utils/fieldValidation');
 
 function getQuestion(getQuestionService) {
   return async(req, res, next) => {
     const hearingId = req.params.hearingId;
     const questionId = req.params.questionId;
     try {
       const response = await getQuestionService(hearingId, questionId);
-      res.render('question.html', {
-        question: {
-          header: response.question_header_text,
-          body: response.question_body_text
+      const question = {
+        hearingId,
+        questionId,
+        header: response.question_header_text,
+        body: response.question_body_text,
+        answer: {
+          value: response.question_answer_text
         }
-      });
+      };
+      req.session.question = question;
+      req.session.save();
```

Comment:
I was just trying to work out if this is required, we have `resave` set to true in the session store config.

Reply:
Yep you're right